runbooks/run-onlyswaps-verifier.md

```diff
@@ -0,0 +1,135 @@
+   ```bash
+   ./onlyswaps-verifier --config path/to/my/config.toml
+   ```
+2. Wait for a few seconds, making sure the verifier starts without errors.
+   Stop the verifier with Ctrl-C.
+3. To run a persistent verifier, we currently have two options: systemd and docker compose.
+   Choose whichever works best, and follow the steps below.
+
+## systemd
+**1. Move onlyswaps-verifier binary to /opt**  
+Run the following commands to move the onlyswaps-verifier binary to `/opt/onlyswaps/onlyswaps-verifier`.
+```bash
+# move onlyswaps-verifier binary to /opt/onlyswaps/onlyswaps-verifier
+mkdir -p /opt/onlyswaps
+mv ./onlyswaps-verifier /opt/onlyswaps/
+```
+
+**2. Move config to /etc**
+Run the following commands to move the onlyswaps-verifier configuration file to `/etc/onlyswaps/verifier.toml`.
+```bash
+# move config to /etc/onlyswaps/verifier.toml
+mkdir -p /etc/onlyswaps
+mv path/to/my/config.toml /etc/onlyswaps/verifier.toml
+
+# adjust verifier.toml permissions
+chmod 640 /etc/onlyswaps/verifier.toml
+```
+
+**3. (Optional) Create a new user**  
+Depending on your setup, you may want/need to create a separate user to run the onlyswaps-verifier service.
+This can be done with the following commands:
+```bash
+# create onlyswaps user
+useradd --system --shell /bin/false --no-create-home onlyswaps
+
+# adjust permissions
+chown onlyswaps:onlyswaps /opt/onlyswaps/onlyswaps-verifier
+chmod +x /opt/onlyswaps/onlyswaps-verifier
+chown root:onlyswaps /etc/onlyswaps/verifier.toml
+```
+
+**4. Create unit file**  
+Create a new unit file under `/etc/systemd/system/onlyswaps-verifier.service`, with the following content:
+```ini
+[Unit]
+Description=onlyswaps-verifier service
+After=network.target
+Wants=network.target
+
+[Service]
+Type=simple
+# TODO: Remove/update it if you didn't create a user
+User=onlyswaps
+Group=onlyswaps
+ExecStart=/opt/onlyswaps/onlyswaps-verifier --config /etc/onlyswaps/verifier.toml
+Restart=always
+RestartSec=10
+StandardOutput=journal
+StandardError=journal
+
+NoNewPrivileges=yes
+PrivateTmp=yes
+ProtectSystem=strict
+ProtectHome=yes
+
+[Install]
+WantedBy=multi-user.target
+```
+
+**5. Run the service**  
+Enable and start the service with the following commands:
+```bash
+# Reload systemd to recognize the new service
+systemctl daemon-reload
+
+# Enable the service to start at boot
+systemctl enable onlyswaps-verifier
+
+# Start the service now
+systemctl start onlyswaps-verifier
+```
+
+Make sure that the service successfully started with `systemctl status onlyswaps-verifier`.
+
+## docker compose
+With docker compose, you may either run the service as root, or with another user (for instructions, you may follow the user creation step shown in [systemd](#systemd)).
+
+**1. Create `docker-compose.yml` configuration**
+1. Save the following `docker-compose.yml` to a location of your choosing:
+   ```yaml
+   services:
+     onlyswaps-verifier:
+       image: ghcr.io/randa-mu/dcipher/onlyswaps-verifier:main-latest
```

Comment:
not sure what the best course of action is here, but we might want to provide specific hashes for people to use given periods. Perhaps we should tag them with our release versions too